lib/Sparql/Client.php

```diff
@@ -51,6 +51,10 @@
  */
 class Client
 {
+    /** application Content type */
+    const CTYPE_SPARQL_UPDATE = 'sparql-update';
```

Comment:
If you're using content-type, better use the full string (application/sparql-update).

Reply:
Done